xarray/core/concat.py

```diff
@@ -34,74 +34,74 @@ def concat(
         unchanged. If dimension is provided as a DataArray or Index, its name
         is used as the dimension to concatenate along and the values are added
         as a coordinate.
-    data_vars : {'minimal', 'different', 'all' or list of str}, optional
+    data_vars : {"minimal", "different", "all"} or list of str, optional
         These data variables will be concatenated together:
-          * 'minimal': Only data variables in which the dimension already
+          * "minimal": Only data variables in which the dimension already
```

Comment:
These don't need a space?

Reply:
not sure, but I guess the space before the colon is only necessary when separating names from types